convert_matrices.py

```diff
@@ -1725,4 +1725,7 @@ def print_projects(title, projects, file=None):
 if __name__ == '__main__':
     logging.basicConfig(format='%(asctime)s %(levelname)s:%(message)s',
                         level=logging.DEBUG)
-    main(Path.cwd() / 'projects')
+
```

Comment:
Too much vertical space. I only use empty lines to separate blocks of multiple statements. A single line rarely qualifies to stand alone, surrounded by empty lines.